Localization/en-US/Mods.PathOfTerraria.Items.hjson

```diff
@@ -784,3 +784,13 @@ SkeletronMap: {
 		'The map smells of cobwebs and metal'
 		'''
 }
+
+LimpidShard: {
+	DisplayName: Limpid Shard
+	Tooltip:
+    	'''
+    	Turns a magic or rare item normal
+    	Hold an item then right click this Shard to make item normal
```

Comment:
I'd replace "to make item normal" with "to turn it into a normal item", the former is weirdly terse.

Reply:
Updated, but slightly different than suggested.
Additionally added some formatting changes for another file.